src/clj/rems/application/model.clj

```diff
@@ -103,6 +104,16 @@
       (update :application/members disj (:application/member event))
       (update :application/past-members conj (:application/member event))))
 
+(defmethod application-base-view :application.event/actor-invited
+  [application event]
+  (-> application
+      (assoc-in [:application/actor-invitations (:invitation/token event)]
+                (select-keys event [:event/actor :application/actor :invitation/role]))))
```

Comment:
invited actors are stored separately from members because we don't want to leak them via the `:application/invited-members` key